src/pages/AllDao.vue

```diff
@@ -86,16 +86,18 @@ export default defineComponent({
 	setup() {
 		const formState = reactive(DaoStore());
 		// Get all daos
-		executeReq(ALL_DAOS_REQ)
+		// Here 1 --> Page number
+		// 5 --> number of entries to show in single fetch
+		executeReq(ALL_DAOS_REQ(1, 5))
```

Comment:
I think we should save `1, 5` in constant.

Reply:
More work has to done here. I will add TODO.